src/components/NouveautesList.jsx

```diff
@@ -1,98 +1,178 @@
-          <button type="button" value="35">
-            Comédie
+          <button type="button" name="35" onClick={this.chosenGenre}>
+            Humour
           </button>
-          <button type="button" value="80">
+          <button type="button" name="80" onClick={this.chosenGenre}>
             Crime
           </button>
-          <button type="button" value="99">
-            Documentaire
-          </button>
-          <button type="button" value="18">
+          <button type="button" name="18" onClick={this.chosenGenre}>
             Drame
           </button>
-          <button type="button" value="10751">
+          <button type="button" name="10751" onClick={this.chosenGenre}>
             Familial
           </button>
-          <button type="button" value="14">
+          <button type="button" name="14" onClick={this.chosenGenre}>
             Fantastique
           </button>
-          <button type="button" value="36">
+          <button type="button" name="36" onClick={this.chosenGenre}>
             Histoire
           </button>
-          <button type="button" value="27">
+          <button type="button" name="27" onClick={this.chosenGenre}>
             Horreur
           </button>
-          <button type="button" value="10402">
+          <button type="button" name="10402" onClick={this.chosenGenre}>
             Musique
           </button>
-          <button type="button" value="9648">
+          <button type="button" name="9648" onClick={this.chosenGenre}>
             Mystère
           </button>
-          <button type="button" value="10749">
-            Romance
+          <button type="button" name="10749" onClick={this.chosenGenre}>
+            Amour
           </button>
-          <button type="button" value="878">
+          <button type="button" name="878" onClick={this.chosenGenre}>
             S.F.
           </button>
-          <button type="button" value="10770">
+          <button type="button" name="10770" onClick={this.chosenGenre}>
             Téléfilm
           </button>
-          <button type="button" value="53">
+          <button type="button" name="53" onClick={this.chosenGenre}>
             Thriller
           </button>
-          <button type="button" value="10752">
+          <button type="button" name="10752" onClick={this.chosenGenre}>
             Guerre
           </button>
-          <button type="button" value="37">
+          <button type="button" name="37" onClick={this.chosenGenre}>
             Western
           </button>
         </div>
-      </div>
-    </article>
-  );
+      </article>
+    );
+  }
 }
 
-NouveautesList.propTypes = {
-  listNewMovies: PropTypes.arrayOf(
-    PropTypes.shape({ title: PropTypes.string, posterPath: PropTypes.string })
-      .isRequired,
-  ),
-};
+// NouveautesList.propTypes = {
+//   listNewMovies: PropTypes.arrayOf(
+//     PropTypes.shape({ title: PropTypes.string, posterPath: PropTypes.string })
+//       .isRequired,
+//   ),
```

Comment:
~~Pourquoi c'est en commentaire ça 🙂 ? Tu as pu committer quand même ?~~

J'ai rien dit, `listNewMovies` est maintenant dans le state, et  non plus passé dans les props. À ce moment là il vaut mieux supprimer les propTypes devenues inutiles, plutôt que de les laisser commentées.

Reply:
Done in this commit : b35987e8475ad6a9f6d5bd84a419112074ed4bf